ethereum-types/src/lib.rs

```diff
@@ -24,7 +24,14 @@ mod uint;
 pub use uint::{U64, U128, U256, U512};
 pub use hash::{H32, H64, H128, H160, H256, H264, H512, H520};
 pub use ethbloom::{Bloom, BloomRef, Input as BloomInput};
-pub use fixed_hash::clean_0x;
+
+/// Cuts away `"0x"` prefix on the given `input` if existing and returns the result.
+pub fn clean_0x(input: &str) -> &str {
```

Comment:
It's baaaaaack! AAAARGHHH! 🥶

Can we deprecate this please?

Reply:
hahaha, yeah I look into it!